### PR TITLE
move storage to firebase 🎉

### LINKED ISSUE #9: Migrate data storage from local storage to firebase
Now that we can authenticate with Firebase, we should migrate data out of local storage and on to the user!

I think its worth discussing how we organize that data. So open to suggestions on this front!

### PR DESCRIPTION
**Pull request by SaraVieira**
closes #9 

Renames local methods
adds loading state in screens
moves files from local storage to firebase

🤯

### CODE REVIEW

File: screens/Reports.js [hunk 1]
```diff
@@ -17,8 +17,23 @@ import {
 } from '../utils/actions';
 import type { CheckIn } from '../utils/types';
 
+const buildData = (dataSet, ratingResult) => ({
```
Comment (kkemple) [inline screens/Reports.js:20]: :heart: :heart: :heart:

**Review by kkemple — APPROVED** [review]
this looks amazing! you got this done so fast 🐎

**Comment by kkemple** [comment]
@SaraVieira i made a few changes after resolving a few conflicts, just letting you know in case you want to review my changes! otherwise this looks great, feel free to merge whenever!

**Comment by SaraVieira** [comment]
All looks awesome ! I'mma gonna merge 😎

About the 🐎 : Nice catch ! I used to work for https://tvg.com so I needed to move as fast as those horses ahah

**Pull request merged**